### PR TITLE
Migrate hasura from Sanity to MDX

### PR DESCRIPTION
**Pull request by alexisintech**
/docs/integrations/databases/hasura

This PR

- migrates this page from Sanity to MDX
- adds links to references
- adds images and optimized them to webp's using Squoosh

Acknowledging that these pages probably need work, and need their code examples updated - but for the scope of "Sanity to MDX Migration", this can be revisited at a later time, with better resources.

For reference, this is what the page looked like before this PR:

https://github.com/clerkinc/clerk-docs/assets/98043211/e616c0be-d9f4-424a-abad-b6ffb709c80a

### CODE REVIEW

File: docs/integrations/databases/hasura.mdx
Comment (BrandonRomano) [inline docs/integrations/databases/hasura.mdx:6]: The H1 of these database pages should probably not just be "Getting Started", and it should include the name of the database in question.  I looked horizontally against other DB pages and they all seem to be starting with just "Getting started".

The reason for this is twofold:

- The first one is user readability perspective; we want to ensure the user is contextualized on the page so they know what they are looking at.  Right now the sidebar is the only thing that contextualizes the user on the page to know what DB they are looking at.  On mobile this sidebar is less prevalent hidden behind a click.
- Secondarily, there may be some SEO impacts if we have a bunch of pages with H1s that are just "Getting started"

I would look at how we approach our quick-start guides, as I think those H1s are well-formed. https://clerk.com/docs/quickstarts/nextjs

File: docs/integrations/databases/hasura.mdx
Comment (BrandonRomano) [inline docs/integrations/databases/hasura.mdx:13]: These images are too big and could be scaled down. I think the max size of container that the images are put in are less than 1000px (I think the breakpoint where the sidebar disappears is where the container is largest).

**Review by BrandonRomano — CHANGES_REQUESTED** [review]
Few suggestions!

File: docs/integrations/databases/hasura.mdx
Comment (alexisintech) [reply docs/integrations/databases/hasura.mdx:6]: Yep, there's a ticket to fix this once these are all merged. Because some are already merged with these H1's, this can be done all at once.
Ticket: https://linear.app/clerk/issue/DOCS-342/fix-integrations-h1s

File: docs/integrations/databases/hasura.mdx
Comment (alexisintech) [reply docs/integrations/databases/hasura.mdx:13]: Sure, these are just the natural width and height on the image properties. I can cut them in half - but I'd appreciate to be educated on the why for this suggestion!

Comment (BrandonRomano) [reply docs/integrations/databases/hasura.mdx:13]: It's largely an optimization issue; we are loading images that are much larger than needed.  This leads to an increased file size that takes longer to load,  but we do not benefit from the increased image size because the browser has to scale the larger image down to fit in its container.

CleanShot 2023-09-22 at 10 48 02@2x

File: docs/integrations/databases/hasura.mdx
Comment (BrandonRomano) [reply docs/integrations/databases/hasura.mdx:6]: Cool cool; happy for this to be deferred, that won't hold up my approval then as long as we have a ticket to address!

File: docs/integrations/databases/hasura.mdx
Comment (alexisintech) [reply docs/integrations/databases/hasura.mdx:13]: Oh yes this part makes sense, but I thought Next/Image handled this 😭 there are so many images that need to be fixed 😭😭😭

Comment (alexisintech) [reply docs/integrations/databases/hasura.mdx:13]: but thank you, this makes a lot of sense I should have thought about this I learned this along time ago!! totally slipped my mind

**Review by BrandonRomano — APPROVED** [review]

**Pull request merged**
